### DISCUSSION TITLE
左边日历排版是竖着的？

### DISCUSSION THREAD

**Discussion by neilwyh** [usememos/memos#353] [General]
![image](https://user-images.githubusercontent.com/4897422/198541739-e33eac31-6a61-402e-b208-dadf489e8570.png)

如上图，这两天开始用，突然发现的

**Comment by YuJianghao** [comment]
应该是模仿 Github 的吧
![image](https://user-images.githubusercontent.com/17293034/198564414-03a6fc67-c670-48d0-8366-8c09bc025731.png)

**Reply by YuJianghao** [reply]
不过感觉和 Github 一样写星期一三五更合适些？如果你也这么想我可以改一下 @boojack
